build/gtsrb/README.md

```diff
@@ -0,0 +1,26 @@
+# Brevitas GTSRB example
+
+This is the binarized CNV topology from the paper [FINN: A Framework for Fast, Scalable Binarized Neural Network Inference](https://arxiv.org/abs/1612.07119) which is trained
+on the [German Traffic Sign Recognition Benchmark (GTSRB)](https://benchmark.ini.rub.de/gtsrb_news.html) dataset.
+
+## Build bitfiles for GTSRB
+
+0. Ensure you have performed the *Setup* steps in the top-level README for setting up the FINN requirements and environment variables.
+
+1. Run the `download-model.sh` script under the `models` directory to download the pretrained QONNX model. You should have e.g. `gtsrb/models/cnv_1w1a_gtsrb.onnx` as a result.
```

Comment:
Maybe leave out "e.g." since the model will be exactly that name. ".../models/cnv_1w1a_gtsrb.onnx"

Reply:
Changed